doc/compiled.json

```diff
@@ -4188,6 +4279,15 @@
           "type": "string"
         }
       },
+      "locale_codes": {
+        "in": "query",
+        "name": "locale_codes",
+        "description": "Locale Code",
+        "required": false,
+        "schema": {
+          "type": "string"
```

Comment:
I'm wondering if this could be actually an array. https://swagger.io/docs/specification/serialization/ (section Query Parameters, `explode:false` should result with a comma separated string in query parameter)

Reply:
It would be the first param of this type in our API, maybe a more broader decision?